sled-hardware/src/lib.rs

```diff
@@ -210,9 +222,12 @@ pub fn detect_cpu_family(log: &Logger) -> sled_hardware_types::CpuFamily {
     info!(
         log,
         "read CPUID leaf 1 to detect CPU family";
-        "values" => ?leaf_1,
-        "family" => family,
-        "model" => model,
+        "leaf1.eax" => format_args!("{:#08x}", leaf_1.eax),
+        "leaf1.ebx" => format_args!("{:#08x}", leaf_1.ebx),
+        "leaf1.ecx" => format_args!("{:#08x}", leaf_1.ecx),
+        "leaf1.edx" => format_args!("{:#08x}", leaf_1.edx),
+        "parsed family" => format_args!("{family:#x}"),
+        "parsed model" => format_args!("{model:#x}"),
```

Comment:
turbo nit (i'm sorry): usually we don't use spaces (or dots) in `slog` field names, since they get turned into JSON fields by the logger --- while those characters are _permitted_ in JSON, i think they're a bit impolite (they might force you to quote the field in `jq` and similar tools when you don't have to otherwise). normally we use underscores for these. not a big deal.